src/basic_memory/schemas/base.py

```diff
@@ -187,10 +187,20 @@ class Entity(BaseModel):
     @property
     def file_path(self):
         """Get the file path for this entity based on its permalink."""
+        # Import here to avoid circular dependency
+        try:
+            from basic_memory.config import app_config
+            use_kebab_case = app_config.filename_format == "kebab-case"
+        except ImportError:
+            # Fallback to original behavior if config not available
+            use_kebab_case = False
```

Comment:
This should probably be 
```suggestion
            use_kebab_case = True
```